doc/common/chapter00.rst

```diff
@@ -0,0 +1,281 @@
+Anuket Project
+==============
+
+Overview
+========
+
+Initially organized early in 2019, the Cloud iNfrastructure Telco Taskforce (CNTT) was created in response to rapid changes in how networking applications are being designed, built and managed, plus a growing recognition of a perceived functional gap between the previous standard infrastructure models and the architectures needed to support Network Function Virtualisation (NFV) applications. Organizationally, the Cloud iNfrastructure Telco Taskforce was jointly hosted by GSMA and the Linux Foundation.
+In 2021, the CNTT and the Open Platform for NFV (OPNFV) projects were merged to form the Anuket project.
+
+Terminology and Glossary
+------------------------
+
+The definition and intent of the terminology used throughout the documents is defined in the `Glossary <./glossary.md>`__.
```

Comment:
since glossary is also converted to rst -- the uri should refer to the rst file

Reply:
Fixed in baaa3ac